cubic-server/nbt.hpp

```diff
@@ -183,6 +182,12 @@ class Compound : public Base
         // Ends the TAG_Compound with a TAG_End
         data.push_back(0);
     }
+
+    constexpr void destroy() override {
+        for (auto i : _value) {
+            delete i;
```

Comment:
I think you need to call destroy before deleting it, cause if you have a nested list or compound, it will not be freed